### DISCUSSION TITLE
I just want simple method argument checking without going down the `typeChecking=basic` painful rabbit hole

### DISCUSSION THREAD

**Discussion by akrherz** [microsoft/pylance-release#6404] [Q&A] [answered]
Sorry to be grumpy, but I just want pylance/pyright to tell me when I make silly mistakes like this
```python
def foo(a):
   ...

foo()
````

As it stands now, I have to enable `python.analysis.typeChecking=basic` to get this problem reported.  Enabling that though causes a mountain of false positives to be generated from the usage of libraries like `pandas` and about everything else under the sun it seems.  Oye.  There must be a simple way? Thank you.

**Comment by erictraut** [comment, marked as answer]
You can keep `typeChecking=off` but enable only the `reportCallIssue` diagnostic check. This might be slightly broader than what you're looking for, but it's pretty close. You could set the severity level to `error` or `warning` depending on how you'd like these diagnostics to appear.

**Reply by akrherz** [reply]
Thank you!  What a relief :)
```json
"python.analysis.diagnosticSeverityOverrides": {
    "reportCallIssue": "warning",
}
```
